packages/console/src/assets/docs/guides/api-python/README.mdx

```diff
@@ -41,12 +41,10 @@ pip install python-jose[ecdsa]
 
 ### Retrieve Logto's OIDC configurations
 
-<p>
 You will need a JWK public key set and the token issuer to verify the signature and source of the received JWS token.
-All the latest public Logto Authorization Configurations can be found at <code>{appendPath(props.endpoint, '/oidc/.well-known/openid-configuration')}</code>.
+All the latest public Logto Authorization Configurations can be found at <Code>{appendPath(props.endpoint, '/oidc/.well-known/openid-configuration')}</Code>.
```

Comment:
Curious why we need to replace <code/> with <Code/> components, would you like to add some explanation in the PR, thanks!

Reply:
i'll find the root cause later